locales/es/translation.json

```diff
@@ -1,1327 +1,1328 @@
+    "The project cannot be copied due to insufficient permission": "El proyecto no puede copiarse por falta de permisos",
+    "The project cannot be deleted due to insufficient permission": "El proyecto no puede eliminarse por falta de permisos",
+    "The project cannot be edited due to insufficient permission": "El proyecto no puede editarse por falta de permisos",
+    "The project cannot be synced because a sync job is currently running": "El proyecto no se puede sincronizar porque se está ejecutando una tarea de sincronización",
+    "The project cannot be synced due to insufficient permission": "El proyecto no se puede sincronizar debido a permisos insuficientes",
+    "The project sync cannot be canceled because it is not running": "La sincronización del proyecto no puede cancelarse porque no se está ejecutando",
+    "The project sync cannot be canceled due to insufficient permission": "La sincronización del proyecto no puede cancelarse por falta de permisos",
+    "The team cannot be deleted due to insufficient permissions.": "El equipo no puede ser eliminado debido a permisos insuficientes.",
+    "The team cannot be edited due to insufficient permissions.": "El equipo no puede ser editado debido a permisos insuficientes.",
+    "The token needed to utilize the SCM URL.": "El token necesario para utilizar la URL SCM.",
+    "The user cannot be deleted due to insufficient permissions.": "El usuario no puede ser eliminado debido a permisos insuficientes.",
+    "The user cannot be edited due to insufficient permissions.": "El usuario no puede ser editado debido a permisos insuficientes.",
+    "There are currently no decision environments": "Actualmente no hay entornos de decisión",
+    "There are currently no inventories added.": "Actualmente no hay inventarios añadidos.",
+    "There are currently no jobs": "Actualmente no hay tareas",
+    "There are currently no projects": "Actualmente no hay proyectos",
+    "There are currently no projects added to your organization.": "Actualmente no hay proyectos añadidos a su organización.",
+    "There are currently no roles added for your organization.": "Actualmente no hay roles añadidos para su organización.",
+    "There are currently no rule audit records": "Actualmente no hay registros de auditoría de reglas",
+    "There are currently no rulebook activations": "Actualmente no hay activaciones del reglamento",
+    "There are currently no teams added to your organization.": "Actualmente no hay equipos añadidos a su organización.",
+    "There are currently no users added.": "Actualmente no hay usuarios añadidos.",
+    "There is currently only one SCM available for use.": "Actualmente sólo hay un MEC disponible para su uso.",
+    "There is currently only one SCM type available for use.": "Actualmente sólo se puede utilizar un tipo de MEC.",
```

Comment:
What is MEC? I think this is a wrong translation. AFAIK, we use the original acronym "SCM"